.editorconfig

```diff
@@ -3,10 +3,13 @@
 
 root = true
 
+[Makefile]
+indent_style = tab
+
 [*]
 end_of_line = LF
 
-[*]
+[*.{hs,lhs}]
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
```

Comment:
Could we move the rule to trim whitespace for all file types? I hope that will no break Makefiles like indent with spaces

Reply:
I tried the change, it doesn't affect Makefiles from what I can see.